src/main/kotlin/com/r3/developers/csdetemplate/MyFirstFlow.kt

```diff
@@ -87,7 +86,7 @@ class MyFirstFlow: RPCStartableFlow {
         // Receive a response from the Responder flow
         val response = session.receive(Message::class.java)
 
-        // The return value of a RPCStartableFlow must always be a String, this string will be passed
+        // The return value of a RPCStartableFlow must always be a String, this string will pass
```

Comment:
Does String need to start with a capital? Just checking

After string, put a full stop before starting next sentence (This string will pass......)